src/core/qgsvectorfilewriter.cpp

```diff
@@ -982,6 +982,48 @@ QMap<QString, QgsVectorFileWriter::MetaData> QgsVectorFileWriter::initMetaData()
                          )
                        );
 
```

Comment:
I remember that at some point you said these options can be queried from OGR. Is this correct or am I confused?